### DISCUSSION TITLE
Mutation: Pass an array as argument

### DISCUSSION THREAD

**Discussion by duranrandy** [rebing/graphql-laravel#912] [Q&A]
I am reading the documentation, and I do not see anywhere how to pass an array as an argument in a mutation, the types I find are String, Int, boolean, Float, ID.

Do you have any suggestions on how to do that? 

thanks in advance

**Comment by mfn** [comment]
You use `Type::listOf(…)` and wrap your type.

Indeed I don't think there's a specific example in the README of this project. To some degree, understanding how https://github.com/webonyx/graphql-php/ , the underlying GraphQL library, works is assumed in places. graphql-library is meant to be a "thin" layer.

**Reply by duranrandy** [reply]
Thank you, I'm going to take a look at it
